bucket/networkmanager.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "2021.1.0",
+    "description": "Tool for managing networks and network troubleshooting",
+    "homepage": "https://borntoberoot.net/NETworkManager/",
+    "license": "GPL-3.0-or-later",
+    "suggest": {
+        ".NET": "dotnet-sdk"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/BornToBeRoot/NETworkManager/releases/download/2021.1.0/NETworkManager_2021.1.0_Archiv.zip",
+            "hash": "425517a996ce52bb2adff05d02537817c954329402aae9f05773b36c61997e71"
+        }
+    },
+    "bin": "NETworkManager.exe",
```

Comment:
What are available parameters?

Reply:
Not sure what do you mean. NETworkManager is graphical app AFAIK does not accept any command line arguments.

Reply:
Then why do you add binary??

Reply:
Because I blindly coped from other manifest not fully understanding contents. Will fix that.